### PR TITLE
Design fix

### PR DESCRIPTION
**Pull request by hemel-cse**
Resolved the issue with the [pull request](https://github.com/Hedayet/css-to-js-object/pull/1). Still for the mobile design need to be done. 

Current look [here](https://pasteboard.co/I54csjE.png).

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  }
+  return styles
+}
+
+const dashedToCamelCase = str => {
+  let ret = ''
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] == '-') {
+      if (i < str.length - 1) {
```
Comment (taman-islam) [inline assets/script.js]: i < str.length-1

File: assets/script.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        ret += str[i + 1].toUpperCase()
```
Comment (taman-islam) [inline assets/script.js]: str[i+1]

File: assets/script.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  for (let k in attrs) {
+    rets.push(dashedToCamelCase(k) + ': "' + attrs[k] + '"')
+  }
+
+  document.getElementById('out').value =
+    stylePrefix + '{{ ' + rets.join(', ') + ' }}'
+  return false
+}
+
+const conquere = styleString => {
```
Comment (taman-islam) [inline assets/script.js]: s/conquere/conquer

**Review by taman-islam — APPROVED** [review]

**Comment by taman-islam** [comment]
@hemel-cse please rebase and I'll merge. thanks.

**Comment by hemel-cse** [comment]
Sorry for late. Working on it.

**Pull request closed without merging**